components/colors/demo/colors.html

```diff
@@ -0,0 +1,149 @@
+
+			<h3>Fluorite</h3>
+			<d2l-demo-snippet>
+				<d2l-color-swatch name="fluorite-plus-2"></d2l-color-swatch>
+				<d2l-color-swatch name="fluorite-plus-1"></d2l-color-swatch>
+				<d2l-color-swatch name="fluorite"></d2l-color-swatch>
+				<d2l-color-swatch name="fluorite-minus-1"></d2l-color-swatch>
+			</d2l-demo-snippet>
+
+			<h3>Tourmaline</h3>
+			<d2l-demo-snippet>
+				<d2l-color-swatch name="tourmaline-plus-2"></d2l-color-swatch>
+				<d2l-color-swatch name="tourmaline-plus-1"></d2l-color-swatch>
+				<d2l-color-swatch name="tourmaline"></d2l-color-swatch>
+				<d2l-color-swatch name="tourmaline-minus-1"></d2l-color-swatch>
+			</d2l-demo-snippet>
+
+			<h3>Cinnabar</h3>
+			<d2l-demo-snippet>
+				<d2l-color-swatch name="cinnabar-plus-2"></d2l-color-swatch>
+				<d2l-color-swatch name="cinnabar-plus-1"></d2l-color-swatch>
+				<d2l-color-swatch name="cinnabar"></d2l-color-swatch>
+				<d2l-color-swatch name="cinnabar-minus-1"></d2l-color-swatch>
+			</d2l-demo-snippet>
+
+			<h3>Carnelian</h3>
+			<d2l-demo-snippet>
+				<d2l-color-swatch name="carnelian-plus-1"></d2l-color-swatch>
+				<d2l-color-swatch name="carnelian"></d2l-color-swatch>
+				<d2l-color-swatch name="carnelian-minus-1"></d2l-color-swatch>
+				<d2l-color-swatch name="carnelian-minus-2"></d2l-color-swatch>
+			</d2l-demo-snippet>
+
+			<h3>Citrine</h3>
+			<d2l-demo-snippet>
+				<d2l-color-swatch name="citrine-plus-1"></d2l-color-swatch>
+				<d2l-color-swatch name="citrine"></d2l-color-swatch>
+				<d2l-color-swatch name="citrine-minus-1"></d2l-color-swatch>
+				<d2l-color-swatch name="citrine-minus-2"></d2l-color-swatch>
+			</d2l-demo-snippet>
+
+			<h3>Topaz</h3>
+			<d2l-demo-snippet>
+				<d2l-color-swatch name="topaz-plus-1"></d2l-color-swatch>
+				<d2l-color-swatch name="topaz"></d2l-color-swatch>
+				<d2l-color-swatch name="topaz-minus-1"></d2l-color-swatch>
+				<d2l-color-swatch name="topaz-minus-2"></d2l-color-swatch>
+			</d2l-demo-snippet>
+
+			<h3>Peridot</h3>
+			<d2l-demo-snippet>
+				<d2l-color-swatch name="peridot-plus-1"></d2l-color-swatch>
+				<d2l-color-swatch name="peridot"></d2l-color-swatch>
+				<d2l-color-swatch name="peridot-minus-1"></d2l-color-swatch>
+				<d2l-color-swatch name="peridot-minus-2"></d2l-color-swatch>
+			</d2l-demo-snippet>
+
+			<h3>Olivine</h3>
+			<d2l-demo-snippet>
+				<d2l-color-swatch name="olivine-plus-1"></d2l-color-swatch>
+				<d2l-color-swatch name="olivine"></d2l-color-swatch>
+				<d2l-color-swatch name="olivine-minus-1"></d2l-color-swatch>
+				<d2l-color-swatch name="olivine-minus-2"></d2l-color-swatch>
+			</d2l-demo-snippet>
+
+			<h3>Malachite</h3>
+			<d2l-demo-snippet>
+				<d2l-color-swatch name="malachite-plus-1"></d2l-color-swatch>
+				<d2l-color-swatch name="malachite"></d2l-color-swatch>
+				<d2l-color-swatch name="malachite-minus-1"></d2l-color-swatch>
+				<d2l-color-swatch name="malachite-minus-2"></d2l-color-swatch>
+			</d2l-demo-snippet>
+
+		</d2l-demo-page>
+
+	</body>
+</html>
```

Comment:
Worth doing a perceptual diff of the full palette?

Reply:
Oh interesting, yeah, it's probably worthwhile.  Easy enough to add.

Reply:
Added visual-diff for the entire palette (one screenshot).  It's the largest screenshot/diff that we've done so far, taller than the view-port.  I made it one screenshot in order to save a few seconds, and it should be easy enough to inspect the differences.  If need-be, we can break this up.

Reply:
... https://s3.ca-central-1.amazonaws.com/visualdiff.gaudi.d2l/screenshots/colors/2019-05-15%2019.52.02.554/report.html

:)

Reply:
Looks awesome!

Reply:
Nice!!